definitions/linux_proc.yaml

```diff
@@ -0,0 +1,186 @@
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/net/ipv*/conf/*/forwarding'
+      - '/proc/sys/net/ipv4/conf/*/mc_forwarding'
+      - '/proc/sys/net/ipv4/ip_forward'
+labels: [Network, System]
+supported_os: [Linux]
+---
+name: NetworkPathFilteringSettings
+doc: States that determine how the system responds to route manipulation.
+urls: ['https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/net/ipv*/conf/*/accept_source_route'
+      - '/proc/sys/net/ipv4/conf/*/rp_filter'
+      - '/proc/sys/net/ipv4/conf/*/log_martians'
+labels: [Network, System]
+supported_os: [Linux]
+---
+name: NetworkRedirectState
+doc: Redirect send/receive states.
+urls: ['https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/net/ipv*/conf/*/accept_redirects'
+      - '/proc/sys/net/ipv4/conf/*/secure_redirects'
+      - '/proc/sys/net/ipv4/conf/*/send_redirects'
+labels: [Network, System]
+supported_os: [Linux]
+---
+name: ProcMounts
+doc: Current mounted filesystems.
+urls: ['https://www.kernel.org/doc/Documentation/filesystems/proc.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/mounts'
+labels: [System]
+supported_os: [Linux]
+---
+name: RestrictedDmesgReadPrivileges
+doc: Restrict whether non-privileged users can read dmesg.
+urls: ['https://www.kernel.org/doc/Documentation/sysctl/kernel.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/dmesg_restrict'
+labels: [System]
+supported_os: [Linux]
+---
+name: RestrictedKernelPointerReadPrivileges
+doc: Memory address obfuscation settings.
+urls: ['https://www.kernel.org/doc/Documentation/sysctl/kernel.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/kptr_restrict'
+labels: [System]
+supported_os: [Linux]
+---
+name: SecureSuidCoreDumps
+doc: Security controls for suid core dumps.
+urls: ['https://www.kernel.org/doc/Documentation/sysctl/fs.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/fs/suid_dumpable'
+labels: [System]
+supported_os: [Linux]
+---
+name: SecureFsLinks
+doc: Security controls to restrict operations on links in world writable directories.
+urls: ['https://www.kernel.org/doc/Documentation/sysctl/fs.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/fs/protected_hardlinks'
+      - '/proc/sys/fs/protected_symlinks'
+labels: [System]
+supported_os: [Linux]
+---
+name: SyncookieState
+doc: Whether the system uses syncookies.
+urls: ['https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt']
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/net/ipv4/tcp_syncookies'
+labels: [Network, System]
+supported_os: [Linux]
+---
+name: ProcSys
```

Comment:
LinuxProcSysHardeningSettings is probably a better name to describe this particular subset of /proc/sys